pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -526,6 +528,9 @@ type CloudConfig struct {
 		// RouteTableID enables using a specific RouteTable
 		RouteTableID string
 
+		// The IAM role to assume when interacting with AWS
```

Comment:
Nit: I believe go comments are supposed to follow "RoleArn is the IAM role..."

>  Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

https://golang.org/doc/effective_go.html

I'm not that strict on it though :-)